eodatasets3/wagl_md_only.py

```diff
@@ -48,12 +48,14 @@ def package(outdir):
 
 
         with h5py.File(str(INDIR.joinpath(WAGL_FNAME)), 'r') as fid:
-            group = fid[GROUP_PATH]
-            img_paths = [ppjoin(group.name, pth) for pth in find(group, 'IMAGE')]
+            img_paths = [ppjoin(fid.name, pth) for pth in find(fid, 'IMAGE')]
 
             for pathname in img_paths:
                 ds = fid[pathname]
 
+                if ds.dtype == 'bool':
```

Comment:
consider using numpy.dtype.name

Reply:
Thanks for tip.  Good pickup!  Explicit type comparison is much safer.